lib/platform/azure/index.ts

```diff
@@ -770,8 +770,8 @@ export function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
 
 export function cleanRepo(): Promise<void> {
   // istanbul ignore if
-  if (config.storage && config.storage.cleanRepo) {
-    config.storage.cleanRepo();
+  if (gitfs && gitfs.cleanRepo) {
```

Comment:
```suggestion
  if (gitfs?.cleanRepo) {
```

Reply:
I think the condition can be removed, as `gitfs` is always defined.

Reply:
The other platforms seemed to have a "isGitInitialized" check here instead, but I also wonder if it can be removed